Metalinter update (ineffassign warning)

Go metalinter was updated and generates new errors on.
- [x] handler/kubeconfig.go:93:8
- [x] controller/cluster/pending.go:31:2
```
handler/kubeconfig.go:93:8:warning: ineffectual assignment to err (ineffassign)
controller/cluster/pending.go:31:2:warning: ineffectual assignment to changedC (ineffassign)
```

--- a/controller/cluster/pending.go
+++ b/controller/cluster/pending.go
@@ -27,8 +27,8 @@ import (
 	"k8s.io/kubernetes/pkg/apis/extensions"
 )
 
-func (cc *clusterController) syncPendingCluster(c *api.Cluster) (*api.Cluster, error) {
-	changedC, err := cc.checkTimeout(c)
+func (cc *clusterController) syncPendingCluster(c *api.Cluster) (changedC *api.Cluster, err error) {
+	_, err = cc.checkTimeout(c)
 	if err != nil {
 		return nil, err
 	}

--- a/handler/kubeconfig.go
+++ b/handler/kubeconfig.go
@@ -91,6 +91,9 @@ func encodeKubeconfig(w http.ResponseWriter, response interface{}) (err error) {
 	}
 
 	ycfg, err := yaml.JSONToYAML(jcfg)
+	if err != nil {
+		return err
+	}
 	_, err = w.Write(ycfg)
 	return err
 }

